packages/main/src/WheelSlider.js

```diff
@@ -312,6 +320,41 @@ class WheelSlider extends UI5Element {
 		return index;
 	}
 
+	_handleWheel(e) {
+		if (!e) {
+			return;
+		}
+
+		e.stopPropagation();
+		e.preventDefault();
+
+		if (e.timeStamp === this._prevWheelTimestamp || !this._expanded) {
+			return;
+		}
+
+		if (e.deltaY > 0) {
+			this._onArrowUp(e);
```

Comment:
Two things can be improved here:
 - Rename arrowUp and arrowDown to something more general, so that they make sense here (and not only in the keyboard handling code). Like f.e. itemUp, itemDown or whatever comes to mind.
- These 2 functions prevent the default event behavior, but actually they're called with 2 different types of events. Sometimes with keyboard events, other times with mouse wheel event. Better to handle the events in their handlers, so stop propagation there, and let these 2 functions only do the logic.